experimental/helper.go

```diff
@@ -0,0 +1,131 @@
+package experimental
```

Comment:
Please add package documentation like https://golang.org/src/go/doc/doc.go?#L5 , (or sort example in https://blog.golang.org/godoc)

Reply:
done